packages/node_modules/@ciscospark/widget-base/src/setup-data-api.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
```

Comment:
does `babel-plugin-lodash` work if you use `_.camelCase()` rather than `import {camelCase} from 'lodash'`